cmd/otelcol/Dockerfile.windows

```diff
@@ -27,6 +27,22 @@ RUN Add-Type -AssemblyName System.IO.Compression.FileSystem ;\
     [System.IO.Compression.ZipFile]::ExtractToDirectory('agent-bundle_windows_amd64.zip', '.')
 RUN Remove-Item agent-bundle_windows_amd64.zip -force
 
+# Download JMX Metric Gatherer
+ARG JMX_METRIC_GATHERER_RELEASE
+RUN New-Item -Path "C:\\" -Name "opt" -ItemType "directory"
+RUN Invoke-WebRequest -Uri "https://repo1.maven.org/maven2/io/opentelemetry/contrib/opentelemetry-jmx-metrics/$env:JMX_METRIC_GATHERER_RELEASE/opentelemetry-jmx-metrics-$env:JMX_METRIC_GATHERER_RELEASE.jar" -Outfile "c:\opt\opentelemetry-java-contrib-jmx-metrics.jar"
+
+ARG JAVA_VERSION="11.0.13_8"
+ARG OPENJDK_BASE_URL="https://github.com/AdoptOpenJDK/openjdk11-upstream-binaries/releases/download"
```

Comment:
The AdoptOpenJDK project is archived.  Should we change it to https://github.com/adoptium/temurin11-binaries/releases, which is what we package in the [linux image and packages](https://github.com/signalfx/splunk-otel-collector/blob/main/internal/signalfx-agent/bundle/Dockerfile#L36)?

Also, if we want to automate the version update and ensure consistency with the linux version, we can try including this in https://github.com/signalfx/splunk-otel-collector/blob/main/.gitlab/update-openjdk.sh#L28.

Reply:
Updated, thanks!